### DISCUSSION TITLE
NEW REVELATION: Compiling is breaking some images

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#504] [Q&A]
Assets
![image](https://github.com/user-attachments/assets/3e8b5702-6a51-4134-b4c2-3e1911291aa6)

Exported assets
![image](https://github.com/user-attachments/assets/2784055a-22ed-4994-8554-e6d36afcdc3b)

I think it's this LibPNG thing
![image](https://github.com/user-attachments/assets/15bb169f-0f87-4744-8d29-88848c239b2b)

**Comment by THEkatinamicrowave** [comment]
yknow I think it's either Lime or OpenFL so I'm rehaxelibbing both of those

**Comment by NeeEoo** [comment]
This is a you issue, please stop opening random discussion and use the discord server
